src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawFigure {
```

Comment:
```suggestion
public interface Drawlable {
```

as we can use this interface not only for Figures